From b34103103cdda0c440dfa76ed8f1ee24702c9d47 Mon Sep 17 00:00:00 2001
From: Alexei Cazacov <alexei.cazacov@qt.io>
Date: Wed, 15 Oct 2025 12:53:49 +0300
Subject: [PATCH 411/553] Doc: Fix instructions on how to use CorePrivate

This commit fixes the cmake instructions for the CorePrivate use.

Fixes: QTBUG-141173
Change-Id: Ie572048c1a45df2369e022ead0e43c6660c30bf5
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
(cherry picked from commit 104ad65a173e27a54650ef0c165f82e2fdcb2301)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/doc/src/qtcore.qdoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/corelib/doc/src/qtcore.qdoc b/src/corelib/doc/src/qtcore.qdoc
index ea65d68da58..38d73870af5 100644
--- a/src/corelib/doc/src/qtcore.qdoc
+++ b/src/corelib/doc/src/qtcore.qdoc
@@ -28,7 +28,7 @@
     private Qt Core APIs:
 
     \badcode
-    find_package(Qt6 REQUIRED COMPONENTS Core)
+    find_package(Qt6 REQUIRED COMPONENTS CorePrivate)
     target_link_libraries(mytarget PRIVATE Qt6::CorePrivate)
     \endcode
 */
-- 
2.51.2

